src/test/java/hw2/ex1/pages/HomePage.java

```diff
@@ -0,0 +1,289 @@
+        }
+
+        //monitorTxt
+        {
+            String monitorTxt = getPageProperty("monitorTxt");
+            waitForElementLocatedBy(driver, By.xpath(monitorTxt));
+            String actual = driver.findElement(By.xpath(monitorTxt)).getText();
+            String expected = "To be multiplatform";
+            Assert.assertEquals(actual, expected);
+        }
+
+        //rocketTxt
+        {
+            String rocketTxt = getPageProperty("rocketTxt");
+            waitForElementLocatedBy(driver, By.xpath(rocketTxt));
+            String actual = driver.findElement(By.xpath(rocketTxt)).getText();
+            String expected = "Already have good base\n" + "(about 20 internal and\n" + "some external projects),\n" + "wish to get more…";
+            Assert.assertEquals(actual, expected);
+        }
+    }
+
+    /**
+     * STEP #8: Assert that there is the iframe with “Frame Button” exist
+     * DATA: -
+     * EXPECTED RESULT: The iframe exists
+     */
+    public void iframeWithButtonExists() {
+        String iframe = getPageProperty("iframe");
+        waitForElementLocatedBy(driver, By.xpath(iframe));
+        String actual = driver.findElement(By.xpath(iframe)).getText();
+        String expected = "<p>Your browser does not support iframes.</p>";
+        Assert.assertEquals(actual, expected);
+    }
+
+    /**
+     * STEP #9: Switch to the iframe and check that there is “Frame Button” in the iframe
+     * DATA: -
+     * EXPECTED RESULT: The “Frame Button” exists
+     */
+
+    public void frameButtonExists() {
+        String iframe = getPageProperty("iframe");
+        waitForElementLocatedBy(driver, By.xpath(iframe));
+        //String windowHandler = driver.getWindowHandle();
+        driver.switchTo().frame(0);
+        String iframeBtn = getPageProperty("iframeBtn");
+        String actual = driver.findElement(By.xpath(iframeBtn)).getAttribute("value");
+        String expected = "Frame Button";
+        Assert.assertEquals(actual, expected);
+    }
+
+    /**
+     * STEP #10: Switch to original window back
+     * DATA: -
+     * EXPECTED RESULT: Driver has focus on the original window
+     */
+    public void switchHomePage() {
+        driver.switchTo().defaultContent();
+    }
+
+    /**
+     * STEP #11: Assert that there are 5 items in the Left Section are displayed and they have proper text
+     * DATA:  “Home”, “Contact form”, “Service”, “Metals & Colors”, “Elements packs”
+     * EXPECTED RESULT: Left section menu items are displayed and have proper text
+     */
+    public void assertLeftSectionItems() {
+        //assert "Home"
+        {
+            String homeLeftSide = getPageProperty("homeLeftSide");
+            waitForElementLocatedBy(driver, By.xpath(homeLeftSide));
+            String actual = driver.findElement(By.xpath(homeLeftSide)).getText();
+            String expected = "Home";
+            Assert.assertEquals(actual, expected);
+        }
```

Comment:
You check items one by one. What will happen if e.g. second element will be missed?